### ISSUE TITLE
apt-get install command fail

### ISSUE THREAD

**Issue by pditommaso** [kubenow/KubeNow#101]
The apt-get install fail are reported below: 

```
$ sudo apt-get install nfs-common
E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)
E: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?
```

The procedure reported [here](http://askubuntu.com/questions/15433/unable-to-lock-the-administration-directory-var-lib-dpkg-is-another-process) didn't help. 

It looks the image (ami-6b427518) has been saved in an inconsistent state. 

ps. let's continue next week :)

**Comment by mcapuccini** [comment]
@pditommaso thanks for reporting, we are going to check and get back to you 🙂

**Comment by andersla** [comment]
I have verified the issue. A workaround is to run `sudo dpkg --configure -a` Lets leave current amazon images like this and make sure next version are not left in this state.

**Comment by andersla** [comment]
Fixed in version 0.2.0
